src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,33 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={classNames('TodoInfo',
+        { 'TodoInfo--completed': completed },)}
```

Comment:
you can make it look better if you want
```suggestion
      className={classNames('TodoInfo',
        { 'TodoInfo--completed': completed },
      )}
```